src/js/smooch.jsx

```diff
@@ -232,4 +232,19 @@ export class Smooch {
     showSettingsNotification() {
         store.dispatch(showSettingsNotification());
     }
+
+    getConversation() {
+        const conversation = store.getState().conversation;
+        const convoNotFoundError = () => {
+            throw new Error('conversation not found');
+        };
+
+        if (conversation) {
+            return Promise.resolve(conversation);
+        }
+
+        return connectFaye()
+            .then(res => res || getConversation())
```

Comment:
don't return `res`, return what's in the store... and a copy of it, not the real object to prevent side effects if they start messing around with it outside of the widget